frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -5,7 +5,12 @@ import { StorageClassResourceKind } from '@console/internal/module/k8s';
 import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
 import './storage-class-dropdown.scss';
 
-const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephAndNoobaStorageProvisioners = [
```

Comment:
```suggestion
const cephAndNoobaaStorageProvisioners = [
```

Reply:
Do we really need this long name? Maybe something like `ocsProvisioners`?

Reply:
makes sense!